### DISCUSSION TITLE
Search for a keyword within the headers

### DISCUSSION THREAD

**Discussion by wlaw868** [provectus/kafka-ui#3389] [Q&A] [answered]
We are using v0.4.0 and currently using the search box to identify a message. That has been a powerful function for us. I wonder if there is capability to search within the headers. If there is, can you guide me how to do that? If not, any timeline for such feature to be delivered or can I raise a PR? Thanks!

**Comment by Haarolean** [comment, marked as answer]
Hi, I suggest using smart filters for that
Also, please update to at least 0.5.0
<img width="760" alt="image" src="https://user-images.githubusercontent.com/1494347/220856133-55177285-e765-432a-9957-a8459aa1bdcc.png">

**Reply by helishah-graphitehealth** [reply]
Hi Haarolean,  I am using Kafka UI version v0.7.2. I want to extract a field `traceId` of type UUID from kafka header but the above syntax is not working. 

I tried adding the below text in `Add filter` and non seem to work.
-  headers["traceId"] == "11111111-1111-1111-a68a-53009f127ca3"
- headers.traceId == "11111111-1111-1111-a68a-53009f127ca3"

I tried replacing double quotes with single and capitalizing `h` of `headers`, even though that shouldn't be the case. I am out of ideas. Can you please help?

**Comment by wlaw868** [comment]
Awesome that works perfectly! Thanks.
